src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+import { Todo } from '../../Type/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id="1"
+      className={classNames(
+        'TodoInfo', { 'TodoInfo--completed': todo.completed },
+      )}
+    >
+
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      <a className="UserInfo" href="mailto:Sincere@april.biz">
```

Comment:
The href attribute value in the anchor tag should be dynamic based on the user's email. Consider using `todo.user.email`.